### DISCUSSION TITLE
请问动作里有向左方向键”←“，我该如何填写SendKeys

### DISCUSSION THREAD

**Discussion by 567-su** [shuax/MouseInc.Settings#359] [Q&A] [answered]
我想添加资源管理器里回退的快捷手势，快捷键是Alt+向左键，我试了”←“、”Left“、“向左键”都不行

**Comment by SoyKoa** [comment, marked as answer]
退格键不是左剪头啊，，想什么呢，["SendKeys", "Backspace"]

**Reply by 567-su** [reply]
谢谢，我想我知道是怎么回事了。在我用Clover打开资源管理器的时候，我的向左手势和向右手势不管用了。我误以为是资源管理器的前进后退快捷键与其他软件不一样（因为我在资源管理器都是用Alt+→/Alt+←），结果我试着把clover关了，然后我的向左向右手势都能用了。
虽然我还是不知道这是为什么，即使我用clover，我用键盘按”Backspace“是有用的，但是我用手势就没有用

**Comment by SoyKoa** [comment]
如果你常用clover，，，就不建议你用MouseInc，，，都是手势类软件，容易冲突，，，clover功能更强大，，，基本包含了所有MouseInc的功能，，，我用MouseInc习惯了，，，所以不用clover，，，

**Reply by 567-su** [reply]
可是clover没有右键手势功能（比如右键向左划就是回退），所以我想用MouseInc，但是我又想要Clover的多标签页功能...
总之还是谢谢你了

**Reply by SoyKoa** [reply]
那很简单啊，，，在MouseInc先设置通用动作，，，然后针对资源管理器我标签页做特定动作

**Reply by 567-su** [reply]
emmmm，你的意思是不用clover吗？可是MouseInc不能让资源管理器像浏览器那样同一个窗口多个标签吧。

**Reply by SoyKoa** [reply]
安装一个QTTabBar就可以让资源管理器像浏览器一样支持多标签

**Reply by 567-su** [reply]
是的，我以前用过QTTabBar，但是时不时总有些卡顿，我就换成了Clover，不过我决定再重新试试QTTabBar，看看QTTabBar和MouseInc有没有冲突，谢谢你的建议😘

**Reply by SoyKoa** [reply]
http://qttabbar.wikidot.com/qttabbar1

QTTabBar和MouseInc 我搭配使用几年了，，一直正常，，，只有电脑重启后，第一次打开“我的电脑”会稍微慢一丢丢，，习惯了

**Reply by 567-su** [reply]
好的好的，感谢了

**Reply by Vermoudol** [reply]
emmm，我也是QT和mouse，但是←的话是Back，资源管理器还是无法起作用啊。添加一个Backspace后仍然没作用...。和上面这位同志一样，键盘按键ALT+←、Backspace是可以的。麻烦hxd看见的话，帮忙解决一下呗。我猜会不会是两个起冲突了

**Reply by SoyKoa** [reply]
![VeryCapture_20220926115005](https://user-images.githubusercontent.com/76244289/192190411-1d2c05db-3d45-4e04-8a98-07b23dc59857.gif)
你们说的是这个意思???我这边能正常操作,,,都几年了,,,一直都可以这样,,,可能是你们装了什么软件冲突了
![image](https://user-images.githubusercontent.com/76244289/192190717-b110838e-90fa-4650-9c68-d3d853c1bbd3.png)

**Reply by Pike-X** [reply]
请问大佬，你的**右键双击**命令是？谢谢分享~

**Reply by SoyKoa** [reply]
> 请问大佬，你的**右键双击**命令是？谢谢分享~

[ [ "SendClick", "left" ], [ "SendClick", "left" ] ]

2次sendclick左键就是双击了

**Reply by Pike-X** [reply]
瑞斯拜克特

**Comment by shuax** [comment]
https://docs.shuax.com/MouseInc/#/faq#%E6%80%8E%E4%B9%88%E6%88%91%E5%9C%A8clover%E4%B8%8A%E4%BD%BF%E7%94%A8%E6%89%8B%E5%8A%BF%E4%B8%8D%E8%83%BD%E5%A4%8D%E5%88%B6%EF%BC%9F
